### PR TITLE
Updated extractText()

### PR DESCRIPTION
**Pull request by Tom-Evers**
Added changes proposed in issue #17

### CODE REVIEW

**Comment by Tom-Evers** [comment]
Some lines contain multiple draw operations, for example if underlined text is drawn text first, underlining ("________") second at the same vertical coordinates.

The toggle 'skip_intertwining_text' will by default skip the next line if intertwining text is detected.
When set to false, it will simply insert text after the previous line.

Indentation is now also properly handled.

**Comment by Tom-Evers** [comment]
It should, yeah, but it has been some time since I worked on this...

The problem is: if the PDF itself has the text in the wrong order but relocated with weird offsets, there's a good chance it'll still mess up the order. Then again, the method that was used before my commit would then still be worse.

**Comment by deven96** [comment]
Good work though mate

**Comment by Tom-Evers** [comment]
Can this be tested/pulled?

**Comment by joegrange** [comment]
Looks like a good improvement.  I'm also having white space issues that this should improve.

**Comment by Tom-Evers** [comment]
It should only improve things, and never break anything that isn't broken already.

Could this be pulled please?

File: PyPDF2/pdf.py [hunk 1]
```diff
@@ -2697,13 +2697,18 @@ def extractText(self, Tj_sep="", TJ_sep=" "):
         # Note: we check all strings are TextStringObjects.  ByteStringObjects
         # are strings where the byte->string encoding was unknown, so adding
         # them to the text here would be gibberish.
+        #
```
Comment (MartinThoma) [inline PyPDF2/pdf.py:2700]:
```suggestion
        #
```

**Comment by MartinThoma** [comment]
@TZanke I just became the maintainer this month - and PyPDF2 is moving again :rocket:

**Comment by MartinThoma** [comment]
It seems like this PR breaks a couple of things. Could you please have a look?

**Comment by MartinThoma** [comment]
This PR addressed #17, but #924 fixed it (+ many other things). Hence I close it.

Thank you for the PR! I hope I can respond quicker in future to such improvements :-)

**Pull request closed without merging**